### ISSUE TITLE
Status messages not reading out to screenreader users

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#579]
## Steps to replicate
* Edit some content
* Select "Save" 

## Expected behaviour
Status messages will read out to screen reader users.

## Observed behaviour

The Status message appears “Status Message Changes have been saved” it does not read out to screen reader users. Due to issue logged above user is at the top of the screen. Assume if the same approach has been used throughout the application that this same issue will also be present for other status messages.

**Comment by j4-m** [comment]
Hi @keelanfh can you tell us what you were testing with. Thanks.

**Comment by j4-m** [comment]
I've done a little research on this and it doesn't appear that [there is a native solution to this](https://github.com/w3c/aria/issues/1360).  Some possible workarounds include:

1. Make the status messages focusable, by giving it a `tabindex` and programmatically focussing that element on page load.
2. Nest the messages in an `aria-live` element, hide (`display: none`) the messages on server render and show on page load.

Any thoughts @msayoung ?
